packages/internal/src/ses-utils.js

```diff
@@ -185,7 +185,10 @@ export const assertAllDefined = obj => {
 };
 
 /**
- * Attenuate `specimen` to only properties allowed by `permit`.
+ * Deeply pick from `specimen` only properties allowed by `permit`.
+ *
+ * If you want to attenuate authority at a defensible boundary for POLA
+ * purposes, see `attenuateOne`.
```

Comment:
Can you explain what is meant by "attenuate authority at a defensible boundary for POLA purposes"? `attenuateOne` returns an Exo while `deepPick` applies a permit and optional transformer to an arbitrary JS object, which seem only loosely related to me but not divergent in terms of behavior of the resulting object's properties (modulo revocability, which would not be expected of `deepPick` anyway).

Reply:
To start with, `deepPick`'s result isn't even hardened by default.

Reply:
Right, and that's important to preserve—it's trivial to harden the output directly, but doing so would preclude its use with inputs that are not compatible with hardening (such as regular expressions and those from external libraries). But I'm asking more precisely about the definitions of "attenuate authority", "defensible boundary", and "POLA purposes", because my naïve reading of those terms applies to both `deepPick` and `attenuateOne` (i.e., both attenuate authority by filtering an input collection of properties [in the latter case, methods] to just those that are appropriate by application of the principle of least authority).

Having written that, I guess "defensible boundary" is the gap that would be established by `harden`?

Reply:
> Having written that, I guess "defensible boundary" is the gap that would be established by harden?

Yes. But more than that, we're moving to a style where exos are the defensible invocation boundary because they bundle in
- only passable may pass by default, propagating that remotables and passable promises are the only invocables that can pass over over these boundaries. As we more towards these remotables themselves being exos, and these passable promises are promises for passable, we start to have a more defensible boundary. Unfortunately, we cannot now ensure that passable promises are promises for passables. This is a hole that will be hard to plug. (I have some ideas. Separate topic.)
- exos are guarded by interface guards, which provide declarative sound type-like input and output validation. The method still has to do its own remaining validation, but that reduces it to a solvable problem.
- anything thrown back over the exo boundary is coerced to a passable error before being rethrown, so throws are not a way to circumvent the boundary.

Having written a whole lot of defensive code at the JS level, if it is low level enough and `harden`ed, then we often succeed. But because JS has so many degrees of freedom beyond passable values, it is no longer a form of defensiveness we should teach or ask others to uphold. Rather, where they intend a defensive boundary, they should switch to an exo.

Reply:
OK, that seems worth elaborating upon here (and likewise in the doc comments for `attenuate`).
```suggestion
 * Note that this function neither hardens its result nor establishes a
 * defensible invocation boundary. For those purposes, particularly inside of
 * vat code,  see `attenuateOne`, which creates an Exo whose methods can be
 * protected by interface guards for type-like input and output validation.
```